extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/boot/FastBootEntityManagerFactoryBuilder.java

```diff
@@ -100,6 +101,12 @@ private PersistenceException persistenceException(String message, Exception caus
                         + " and SSL was not disabled automatically for your driver.";
                 break;
             }
+
+            if (t instanceof CommandAcceptanceException) {
+                message = "Invalid import file. Make sure your statements are valid and properly separated by a semi-colon.";
+                break;
+            }
```

Comment:
So in the end, you managed to do it?

Reply:
Works only if the `generation.halt-on-error` is set to true. See `application-invalid-multiline-test.properties", "application.properties` and the associated test. I am still thinking about this.

Reply:
> Works only if the `generation.halt-on-error` is set to true. See `application-invalid-multiline-test.properties` and the associated test. I am still thinking about this.

What I meant to say is, the message will be displayed when
```
quarkus.hibernate-orm.database.generation.halt-on-error=true
```
config is set to true (only why to intercept the exception). Otherwise whatever went wrong regarding load script file will be logged and the application starts successfully (normal `ExceptionHandler` behaviour) but without our descriptive message related to multiline.